doc/syntax.md

```diff
@@ -53,45 +334,84 @@ end
   `<name>`. The content will be translated into a single neuron model
   that can be instantiated in NEST using `nest.Create(<name>)`. This
   is the top-level block of a model specification and all following
-  blocks are contained in this block.
-
-`parameters`
-: blabla
-
-`state`
-: blabla
-
-`internals`
-: blabla
-
-`equations`
-: blabla
-
-`input`
-: blabla
-
-`output`
-: blabla
-
-`update`
-: blabla
-
-`function`
-: blabla
+  blocks are contained in this block. The block is closed with the `end` keyword.
+
+`parameters:`
+:  This block is composed of a list with variable declarations. This block is
+supposed to contain variables
+which remain constant during the concrete simulation, but they can vary among
+different simulations or instantiations neuron of the same neuron. The block
+is closed with the `end` keyword.
+
+`state:`
+: This block is composed of a list with variable declarations. These variables are supposed to
+model the time dependent  state of the neuron. Only variables form this block can be refined with
+differential equations. The block terminates with the `end` keyword.
+
+`internals:`
+:  This block is composed of a list with variable declarations. These variables
+are supposed to be constant during the simulation run. Therefore, its initialization
```

Comment:
They are constant. I have good reasons to do them constant. If you do want mutable variables, define them in directly in the update block. Otherwise, I cannot do some constant folding optimizations and NESTML code will not meet performance of the handwritten code.